tests/shunit/data/bad_i18n_comment_1.py

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Although the title of the file explains that this is a bad comment, I think it a comment explaining why this example is wrong would be a good idea. This comment should be added to all the `tests/shunit/data` files.

Reply:
done